### PR TITLE
update android app switch example

### PR DESCRIPTION
**Pull request by cekrem**

### CODE REVIEW

File: vipps-ecom-api.md [hunk 1]
```diff
@@ -1810,24 +1810,43 @@ API when the payment is created ("url": "vipps://?token=eyJraWQiOiJqd3RrZXki..")
...
+            val packageInfo = pm.getPackageInfo("no.dnb.vipps", PackageManager.GET_ACTIVITIES)
+
+            if (versionCompare(packageInfo.versionName, vippsMinVersion)) {
+                val intent = Intent(Intent.ACTION_VIEW, Uri.parse(deeplinkUrl))
+                startActivityForResult(intent, vippsRequestCode)
+            } else {
+                // Notify user to download the latest version of the Vipps app
+            }
+        } catch (e: PackageManager.NameNotFoundException) {
+            // No Vipps app! Open play store page.
```
Comment (cloveras) [inline vipps-ecom-api.md:1840]: "Open the Vipps page on Google Play"?

File: vipps-ecom-api.md [hunk 2]
```diff
@@ -1810,24 +1810,43 @@ API when the payment is created ("url": "vipps://?token=eyJraWQiOiJqd3RrZXki..")
...
+
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+
+class MyActivity : Activity() {
+    companion object {
+        const val vippsRequestCode: Int = TODO("insert custom request code")
+        const val vippsMinVersion: String = TODO("insert min version")
```
Comment (cloveras) [inline vipps-ecom-api.md:1824]: This seems quite handy! We should recommend that merchants specify this to make sure the user has the correct version. Maybe add a line of tips/info for that?

File: vipps-ecom-api.md [hunk 3]
```diff
@@ -1810,24 +1810,43 @@ API when the payment is created ("url": "vipps://?token=eyJraWQiOiJqd3RrZXki..")
...
+package com.example.app
+
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+
+class MyActivity : Activity() {
+    companion object {
+        const val vippsRequestCode: Int = TODO("insert custom request code")
```
Comment (cloveras) [inline vipps-ecom-api.md:1823]: What's this for? Do all Android developers know it?

File: vipps-ecom-api.md (hunk 2 above)
Comment (alfthomas) [reply vipps-ecom-api.md:1824]: It is handy, but it's something Vipps will usually handle by forcing updates when needed. Maybe it is better to leave it out to keep the example as simple as possible? Or at least comment that this is something the merchant should usually not set.

File: vipps-ecom-api.md (hunk 3 above)
Comment (cekrem) [reply vipps-ecom-api.md:1823]: "this" meaning the activity class or the companion object? Or the request code? I'm not sure how familiar Android developers are with the latter, but it's nothing we've introduced now. In the old example it was "just there", not being defined anywhere in particular. (`1821: startActivityForResult(intent, requestCode);`)

File: vipps-ecom-api.md (hunk 2 above)
Comment (cekrem) [reply vipps-ecom-api.md:1824]: Sure, let's remove the version check 👍

File: vipps-ecom-api.md (hunk 1 above)
Comment (cekrem) [reply vipps-ecom-api.md]: 👍

**Review by cloveras — APPROVED** [review]

**Review by cloveras — APPROVED** [review]

**Pull request merged**